terminus-terminal/src/index.ts

```diff
@@ -156,9 +156,17 @@ export default class TerminalModule { // eslint-disable-line @typescript-eslint/
             }
         })
         if (config.store.terminal.autoOpen) {
-            app.ready$.subscribe(() => {
-                terminal.openTab()
-            })
+
+            let argv = require('electron').remote.process.argv;
+            if (argv[0].includes('node')) {
+                argv = argv.slice(1)
+            }
+
+            if(require('yargs').parse(argv.slice(1))._[0] !== "open"){
```

Comment:
Isn't this basically `if (argv[1] === 'open')`?

Reply:
Not necessarily. Take for example either running from CLI or starting with a shortcut

`Terminus.exe --debug open "C:\all-the-stuff"`

Here we'd see argv[1] be `--debug`. But if we use yargs.parse and look at `._[]` we will see the first "command" that was used.